.circleci/config.yml

```diff
@@ -1,26 +1,26 @@
 version: 2.1
+
 orbs:
-  elixir: membraneframework/elixir@1
+  elixir: membraneframework/elixir@dev:94bbc77eef4293223822a2f7f31b136ac0833b1f
```

Comment:
Are you sure we want this?

Reply:
I'm experimenting with using non-membrane images where they're not required https://github.com/membraneframework/membrane_core/issues/701

Reply:
Remember to fix it before merging